src/ui/main-window/ThumbnailView.swift

```diff
@@ -95,6 +95,8 @@ class ThumbnailView: NSStackView {
                         yOffset += ThumbnailView.windowsControlSize + ThumbnailView.windowsControlSpacing
                     }
                 }
+                // Force the icon to repaint
+                icon.display()
```

Comment:
Why is this necessary? What was failing before we added this new call?

Reply:
If this modification is not added, an issue arises: after the user clik the fullscrren button to enters full screen, the fullscreen button does not redraw, resulting in the user still seeing the button as if it were in fullscreen mode, rather than the defullscreen button. To make this issue clearer to you, I will open a new MR, so you can see the problem and the solution more clearly.